src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -62,7 +63,7 @@ export class DotHttpErrorManagerService {
         return of(result);
     }
 
-    private callErrorHandler(response: Response): boolean {
+    private callErrorHandler<T>(response: HttpResponse<T>): boolean {
```

Comment:
T here are just placeholders I think, we are not passing anywhere.

Reply:
yes it is a placeholder, in line

https://github.com/dotCMS/core-web/blob/d4202844f4f10011f925b09690dce7df8e36667d/src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts#L52

an error response of type `HttpResponse` is being passed